test/com/facebook/buck/jvm/kotlin/KotlinLibraryIntegrationTest.java

```diff
@@ -161,6 +161,12 @@ public void shouldFailToCompileInvalidKotlinCode() {
     buildResult.assertFailure();
   }
 
+  @Test
+  public void shouldCompileKotlinCodeWithBackticks() {
+    ProcessResult buildResult = workspace.runBuckCommand("build", "//com/example/backticks:example");
```

Comment:
The general pattern here is to just do `workspace.runBuckBuild("//com/example/backticks:example").assertSuccess();`